tests/Doctrine/Tests/ORM/Functional/Ticket/GH7684Test.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\ORM\Mapping\MappingException;
+use Doctrine\Tests\ORM\Functional\DatabaseDriverTestCase;
+
+/**
+ * Verifies that associations/columns with an inline '_id' get named properly
+ * 
+ * Github issue: 7684
+ */
+class GH7684 extends DatabaseDriverTestCase
```

Comment:
Where are other `DatabaseDriverTest` implementations located?

Reply:
(asking because we may get to structure this better, now that we know that the `DatabaseDriver` already has existing tests in place)

Reply:
Should I create a test in `DatabaseDriverTests.php` something like the existing `testLoadMetadataWithForeignKeyFromDatabase` ?